packages/compartment-mapper/src/types.js

```diff
@@ -168,12 +169,18 @@ export const moduleJSDocTypes = true;
  * @returns {string} moduleSpecifier
  */
 
+/**
+ * @callback ShouldDeferError
+ * @param {Language | undefined} language
+ * @returns {boolean}
+ */
+
 /**
  * @callback ImportHookMaker
  * @param {string} packageLocation
  * @param {string} packageName
  * @param {ParseFn} parse
- * @param {Record<string, Object>} exitModules
```

Comment:
exitModules was added (and is used) as input elsewhere, This line seems to be a mistake.

Reply:
You are correct that this code probably died in a refactor. Thanks.